jsk_2016_01_baxter_apc/launch/main.launch

```diff
@@ -14,15 +14,28 @@
     </rosparam>
   </node>
 
+  <!-- set work_order rosparam -->
+  <node name="set_work_order_param"
+        pkg="jsk_2016_01_baxter_apc" type="json_to_rosparam.py">
+    <remap from="~param" to="/work_order" />
+    <rosparam subst_value="true">
+      json: $(arg json)
+      key: work_order
+    </rosparam>
+  </node>
+
   <!-- work order data -->
-  <node pkg="jsk_2016_01_baxter_apc" type="work_order.py" name="work_order" respawn="true">
+  <node pkg="jsk_2016_01_baxter_apc" type="work_order.py" name="work_order_server" respawn="true">
+    <remap from="~param" to="/work_order" />
```

Comment:
it does not detect change of rosparam, actually.
i just add GUI for /work_order.